docs/tutorials/generate-custom-test-data.rst

```diff
@@ -0,0 +1,120 @@
+.. _ksql_generate-custom-test-data:
+
+Generate custom test data by using the ksql-datagen tool
+========================================================
+
+KSQL enables generating test data that complies with a custom schema
+that you define. 
+
+To generate test data, create an Apache Avro schema and pass it to the
+command-line tool, ``ksql-datagen``, which generates random data according
+to the schema you provide.
+
+**Prerequisites:** 
+
+- :ref:`Confluent Platform <installation>` is installed and running.
+  This installation includes a Kafka broker, KSQL, |c3-short|, |zk|,
+  Schema Registry, REST Proxy, and Kafka Connect.
+- If you installed |cp| via TAR or ZIP, navigate to the installation
+  directory. The paths and commands used throughout this tutorial assume
+  that you're in this installation directory.
+- Java: Minimum version 1.8. Install Oracle Java JRE or JDK >= 1.8 on your

```

Comment:
in the future we'll want to use the java include for this piece of text, eventually (i.e. no action now). i'm adding this with my in-progress installation doc revamp.